Dockerfile

```diff
@@ -0,0 +1,43 @@
+#!/bin/echo docker build . -f
+# -*- coding: utf-8 -*-
+# SPDX-License-Identifier: MPL-2.0
```

Comment:
The rest of the project is licensed under Apache 2.0, is it ok to have a single file under a different license?
CC: @bettio

Reply:
mistake let me align